react-native/src/mobile/status_im/react_native/js_dependencies.cljs

```diff
@@ -39,6 +39,7 @@
 (def react-native-screens   (js-require/js-require "react-native-screens"))
 (def mail-class             (js-require/js-require  "react-native-mail"))
 (defn react-native-mail []  (.-default (mail-class)))
+(def async-storage          (.-AsyncStorage react-native))
```

Comment:
Maybe makes sense to use https://github.com/react-native-community/async-storage, as core RN's AsyncStorage is deprecated.

Reply:
yes because we move to RN 60

Reply:
@flexsurfer changed to @react-native-community/async-storage